tests/RoundRobinSet.js

```diff
@@ -0,0 +1,20 @@
+module.exports = RoundRobinSet;
+
+function RoundRobinSet (arr) {
+  this._current = 0;
+  var newArr = arr.slice(0);
+  for (var i = 0, l = newArr.length; i < l; i++) {
+    newArr[i]._rrindex = i;
+  }
+  Object.defineProperty(this,"items",{ value: newArr });
```

Comment:
If you're using `Object.defineProperty()` because `writable` is false by default, maybe you should still specify anyway, so things are clear? Or something like 

```js
function RoundRobinSet(arr) {
  this._current = 0

  Object.defineProperty(this, 'items', {
    get: () => arr.slice().map((e, i)=> Object.assign({_rrindex: i }))
  })
}
```